### PR TITLE
article: Checkly

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  description: "Use Playwright to run your status checks in the cloud to ensure the high availability of your web applications.",
+  author: "Max Schmitt",
+  date: "2020-09-23",
+  tags: ["checkly", "monitoring", "playwright"],
+  hidden: true
+}
+
+## Introduction
+
+Having a safe, functional, and feature-rich configured monitoring and status checks in place is nowadays needed to ensure a fast error reporting and by that a good uptime so your SLA is getting respected. The typical monitoring solutions are e.g. Grafana which would be feature-rich but has no good native integration for launching browsers to emulate a real user. Also, you need to host it yourself which is mostly for monitoring solutions not advised (or you need a HA installation in another data center or cloud provider).
```
Comment (ragog) [inline src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx:12]: My suggestion would be to break down the first phrase, or to rephrase to allow for better punctuation.
One example could be: "Nowadays, catching issues with your service early on is paramount to ensuring that your SLAs are being met. For this reason, having a reliable and feature-rich active monitoring solution in place is a must."

File: src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+To fix these issues, Checkly has created a product to provide developer-friendly status checks, which can be used to launch browsers in the cloud. By using it you also don't need to spin up an own instance in your infrastructure since Checkly is a SaaS (Software as a Service) product and they host it for you. They even provide a full public dashboard for your customers. Starting from today they provide a full Playwright integration.
```
Comment (ragog) [inline src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx:14]: Suggestion: You could further explain what the dashboards do, i.e. show end-users what the status of the service is (through Checkly's check results)

File: src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+<img src="/blog/status-check-monitoring-with-playwright-on-checkly/checkly-home-dashboard.png" style={{maxWidth: "80%"}}/>
+
+## Usage
+
+There are two types of checks which you can use. API Checks to validate a given response body, status code, or schemas like JSON or GraphQL or browser-based tests on which we'll focus on this blog post. Browser-based tests are running a headless Chromium instance in the cloud to validate e.g. that your production login is working. Other common alternative scenarios would be:
```
Comment (ragog) [inline src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx:20]: "Check" appears first as uncapitalised then capitalised. I'd advise sticking to one style.

File: src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+await page.goto("https://playwright.tech");
+
+assert.equal(await page.title(), "Everything about the Playwright framework - Playwright Community 🎭");
+await page.waitForSelector("text=Playwright Community");
+await page.waitForSelector("text=Try Playwright");
+assert.isAbove(await page.$$eval("article", elements => elements.length), 8)
+await browser.close();
+```
+
+In this example which we're ensuring the following checks:
```
Comment (ragog) [inline src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx:43]: I think "which" is extra here.

File: src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+- Assert that the title has the expected value
+- Assert that "Playwright Community" is on the page
+- Assert that "Try Playwright" is on the page
+- Assert that at least 8 articles are on the page
+
+If one of them is failing, Checkly will automatically treat this as a failing test and alert the user by e.g. Slack, PagerDuty, SMS, or Email.
+
+## Integrations
```
Comment (ragog) [inline src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx:52]: Suggestion: mention Terraform. It is one of the most used integrations and allows (through the Checkly Terraform Provider) to automatically manage large numbers of checks - a gamechanger when large APIs and hundreds of checks are involved. https://github.com/checkly/terraform-provider-checkly

File: src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+<img src="/blog/status-check-monitoring-with-playwright-on-checkly/github-pr.png" style={{maxWidth: "min(100%, 500px)"}}/>
+
+## Summary
+
+Checkly provides an easy way of adding either status checks or entire end-to-end tests to your web applications which have now also Playwright support available. We at the Playwright Community are using it to ensure that the services which we publicly expose are on one hand high-available and on the other by that fully-functional when e.g. upgrading dependencies.
+
+We're looking forward to how Checkly will make their monitoring solution even more accessible for developers with e.g. versioned code, an integrated Monaco editor with better auto-completion, support for custom NPM modules or a better debugging experience. We would recommend giving it a try and have not to worry about where to run your status checks or end-to-end tests and benefit from their simplicity.
```
Comment (ragog) [inline src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx:64]: If you think folks might be interesting in seeing how we are progressing on the items you have mentioned, you could point them to our public github roadmap: https://github.com/checkly/public-roadmap

**Review by ragog — COMMENTED** [review]
Great article! I left some suggestions - feel free to pick and choose.

File: src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx (hunk 1 above)
Comment (mxschmitt) [reply src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx:12]: Good point. thx

File: src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx (hunk 3 above)
Comment (mxschmitt) [reply src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx:20]: Good catch, will use the lower case.

**Pull request merged**